tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1005 @@
+                <option value="0">0</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+              </param>
+              <param name="order_by" type="select" label="Order by" help="Field to use for ordering the results">
+                <option value="none" selected="true">No specific order</option>
+                <option value="size" >Order by size (Ko)</option>
+                <option value="collection" >Order by collection</option>
+                <option value="object" >Order by name</option>
+              </param>
+            </when>
+            <when value="raw_query">
+              <param name="table" type="text" label="Table name" help="Name of the table you want to query (FROM clause)" required="True" />
+              <section name="where_clause" title="WHERE clause (field = value)" expanded="false">
+                <param name="where_field" type="text" label="WHERE field" help="WHERE **field** = value" />
+                <param name="where_condition" type="text" label="WHERE value" help="WHERE field = **value**" />
+              </section>
+              <param name="url_field" type="text" label="Url field" help="Table field containing the url of the file to download" required="True" />
+            </when>
+          </conditional>
+        </section>
+        <param name="number_of_files" type="integer" value="1" label="Number of files or urls to download" help="Min = 1 Max = 10" />
+        <param name="csv_file" type="boolean" truevalue="true" falsevalue="false" label="Download urls as csv"/>
+        <param name="image_file" type="boolean" truevalue="true" falsevalue="false" label="Download files"/>
+        <param name="html_file" type="boolean" truevalue="true" falsevalue="false" label="Preview urls in extended html"/>
+        <param name="basic_html_file" type="boolean" truevalue="true" falsevalue="false" label="Preview urls in basic html"/>
+    </inputs>
+    <outputs>
+        <data name="output" format="fits" label="${tool.name} -> File from ${archiveselection.archive_type} search:" >
+          <discover_datasets pattern="__designation_and_ext__" directory="fits" visible="true" format="fits" />
+          <filter>image_file == True</filter>
+        </data>
+        <data name="output_csv" format="txt" label="${tool.name} -> CSV Manifest:">
+          <filter>csv_file == True</filter>
+        </data>
+        <data name="output_html" format="html" label="${tool.name} -> HTML Preview:">
+          <filter>html_file == True</filter>
+        </data>
+        <data name="output_basic_html" format="html" label="${tool.name} -> Basic HTML Preview:">
+          <filter>basic_html_file == True</filter>
+        </data>
+        <data name="output_error" format="txt" label="${tool.name} -> Error Log:"></data>
```

Comment:
It's not nice that it shows "Error Log" at start of every request.

It may be that it's best to just write to `stdout` and then see it in tool execution if needed.

Or/and, we could do like this:

```suggestion
        <data name="output_error" format="txt" label="${tool.name} -> Query summary:"></data>
```

Reply:
Goal was to provide user with more information about why the query might have failed but yes Query summary sounds better and I can also write the error to stdout in addition no problem

Reply:
If proposed change is ok for you, you can just `commit suggestion` without pushing anything else. I will just do it in this case, but see other suggestions.